index.html

```diff
@@ -12,64 +12,73 @@
 
 <body>
     <section class="container">
-        <header>
-           <div class="logo"> <a>Dorcas</a></div>
-            <nav class="Union">
-                <ul>
-                    <li><a href="#portofolio" target="_blank">Portfolio</a></li>
-                    <li><a href="#about" target="_blank">About</a></li>
-                    <li><a href="#contact" target="_blank">Contact</a></li>
-                </ul>
-                <i class="fa fa-list-squares"></i>
-            </nav>
-        </header>
-
-        <main class="content">
-            <section class="subcontent">
-                <h1>
-                    I'm Dorcas
-                    Glad to see
-                    you!
-                </h1>
-                <p>
-                    I’m a software developer! I can help you
-                    build a product , feature or website Look
-                    through some of my work
-                    and experience! If you like what you see and
-                    have a project you need coded, don’t
-                    hestiate to contact me.
-                </p>
-
```

Comment:
- [ ] [OPTIONAL] Kindly try to make the project start from the beginning of the page this will make the project nice and like the Figma design 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/224804166-cc7355e3-ecc7-4b9a-a661-0b73b3bc0feb.png)   |  ![image](https://user-images.githubusercontent.com/83360636/224804232-08bf8af8-0ce0-47a6-97b2-fe8151faf90a.png)  |